src/measure_theory/group/measure.lean

```diff
@@ -3,10 +3,7 @@ Copyright (c) 2020 Floris van Doorn. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn
 -/
-import dynamics.ergodic.measure_preserving
-import measure_theory.measure.regular
-import measure_theory.group.measurable_equiv
-import measure_theory.measure.open_pos
+import measure_theory.group.action
```

Comment:
What's going on with the removed imports here? Are they related to the changes in the PR?

Reply:
I just minimised the imports given that many are included in `measure_theory.group.action`